src/config.hpp

```diff
@@ -3,26 +3,25 @@
 #include <vector>
 
 ///
-/// Dimensions of the screenshots that will be displayed.
-///
+/// Dimensions of the screenshots that will be displayed.  /
 /// Specifying width will stack desktops vertically.
 /// Specifying height will stack desktops horizontally.
 ///
 /// One dimension *must* be set to zero and will be calculated at runtime.
 ///
 /// Actual window width will be equal to dexpo_width + (dexpo_padding * 2)
 ///
-const int dexpo_width = 200;
-const int dexpo_height = 0;
+const int dexpo_width = 0;
+const int dexpo_height = 250;
 
 const uint16_t dexpo_padding = 5; ///< Padding around the screenshots
 
 const uint16_t dexpo_hlwidth = 3; ///< Width of the preselected desktop border
 const uint32_t dexpo_hlcolor = 0XFF662C28; ///< Color of the preselection border
 const uint32_t dexpo_bgcolor = 0xAA525252; ///< Background color. TODO Alpha
 
-const int16_t dexpo_x = 0; ///< X coordinate of window's top left corner
-const int16_t dexpo_y = 0; ///< Y coordinate of window's top left corner
+const int16_t dexpo_x = 1080; ///< X coordinate of window's top left corner
+const int16_t dexpo_y = 20;   ///< Y coordinate of window's top left corner
```

Comment:
Why is it 1080 and 20?

Reply:
Fuck I accidentally pushed in my config
Will reset it in the next commit